### PR TITLE
Fully working comment system

### PR DESCRIPTION
**Pull request by duquej**
comments

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -33,10 +37,23 @@
...
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment");
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    ArrayList<Comment> comments = new ArrayList<>();
+    for (Entity entity : results.asIterable()){
+      String message = (String) entity.getProperty("comment");
```
Comment (KevinWorkman) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Totally optional topic you can research if you're feeling like an extra challenge: I think Datastore might have a limit for how long a String can be? What happens if you try to post a really long comment?

(Like I said, this is totally optional! Feel free to leave this for later or to disregard it entirely.)

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -44,16 +44,19 @@ async function getMessages(){
   const response = await fetch('/data');
   const json = await response.json();
   const container = document.getElementById('message-container');
-  container.appendChild(createListElement(json[0]));
-  container.appendChild(createListElement(json[1]));
-  container.appendChild(createListElement(json[2]));
+  json.forEach(function(comment){
```
Comment (KevinWorkman) [inline portfolio/src/main/webapp/script.js]: ES6 introduced the concept of arrow functions, which would be a little bit shorter (and more JavaScript-y). Something like this:

```
json.forEach((comment) => {
  // your code here
});
```

This is pretty much just syntactic sugar in this case though, so it's up to you which you like best.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -55,6 +55,25 @@ header img {
...
+.comments{
+      overflow-y: auto;
+      height: 300px;
+      margin: 100px 0 100px 100px;
+
+
+}
+
+
+
```
Comment (KevinWorkman) [inline portfolio/src/main/webapp/style.css]: nitpick: you could get rid of these extra newlines

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -55,6 +55,25 @@ header img {
 }
 
 
+.comment {
+       border: thin solid white;
```
Comment (KevinWorkman) [inline portfolio/src/main/webapp/style.css]: Can you add a screenshot in the pull request description? I'm curious to see what this looks like!

Comment (duquej) [reply portfolio/src/main/webapp/style.css]: done!

**Pull request merged**